index.html

```diff
@@ -0,0 +1,323 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta
+        http-equiv="X-UA-Compatible"
+        content="IE=edge"
+    >
+    <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1.0"
+    >
+    <title>Document</title>
+    <link
+        rel="preconnect"
+        href="https://fonts.googleapis.com"
+    >
+    <link
+        rel="preconnect"
+        href="https://fonts.gstatic.com"
+        crossorigin
+    >
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap"
+        rel="stylesheet"
+    >
+    <link
+        rel="stylesheet"
+        href="./style/reset.css"
+    >
+    <link
+        rel="stylesheet"
+        href="style/style.css"
+    >
+</head>
+
+<body>
+
+    <header>
+        <section
+            id="header"
+            class="header"
+        >
```

Comment:
Trochę dziwnie to wygląda jak mamy `header` ale to akurat element `section` ma taką klasę - może to być bardzo mylące :P